File: checker_test/ac/submission.scala
import scala.io.StdIn._

object Test {
  def main(args: Array[String]): Unit = {
    val n = readInt()
    
    val arr = readLine().split(" ").map(_.toInt)

    val max = arr.max
    val maxIndex = arr.indexOf(max)

    println(maxIndex)
  }
}

File: checker_test/ac/submission.kt
fun main(){
    val n = readLine()!!.toInt()
    val array  = readLine()!!.split(" ").map { it.toInt() }.toTypedArray()

    var max = 0
    for (i in 0 until n){
        if (array[i] > max){
            max = array[i]
        }
    }
    val idx = array.indexOf(max)

    println("$idx")
}